### PR TITLE
more docker aliases

### PR DESCRIPTION
**Pull request by matthewadams**

### CODE REVIEW

File: aliases/available/docker.aliases.bash [hunk 1]
```diff
@@ -1,6 +1,8 @@
 cite 'about-alias'
 about-alias 'docker abbreviations'
 
+alias d='docker'
```
[the remark this answers is not in the document]
Comment (matthewadams) [reply aliases/available/docker.aliases.bash]: Yeah wondered why docker was “dk”.  Ignore this one.

**Comment by matthewadams** [comment]
Well, alpine & Ubuntu are so common that I fell they should be included.  I’d be ok to only include alpine.  Do what you want.  I’m ok with whatever.

**Comment by matthewadams** [comment]
Sure np

**Comment by nwinkler** [comment]
Looks good - thanks!

**Pull request merged**